Lib/fontTools/misc/visitor.py

```diff
@@ -85,20 +94,40 @@ def visitObject(self, obj, *args, **kwargs):
             self.visitAttr(obj, key, value, *args, **kwargs)
 
     def visitAttr(self, obj, attr, value, *args, **kwargs):
+        """Called to visit an attribute of an object."""
         self.visit(value, *args, **kwargs)
 
     def visitList(self, obj, *args, **kwargs):
+        """Called to visit any value that is a list."""
```

Comment:
i wonder if we ever need a visitTuple as well.. well, I guess one can always register a custom visit function for tuples if needed.

Reply:
I figured tuples are more often a leaf value than a container. But yeah, they can be registered as a type.